CODEOWNERS

```diff
@@ -374,6 +374,7 @@
 /dts/arm/nxp/                             @MaureenHelm @mmahadevan108 @dleach02
 /dts/arm/microchip/                       @franciscomunoz @albertofloyd @scottwcpg
 /dts/arm/silabs/efm32_pg_1b.dtsi          @rdmeneze
+/dts/arm/seagate/                         @mbittan @simonguinot
```

Comment:
This is an unprecedented step since the SoC is not made by Seagate.

Have you evaluated using board revisions instead? That seems like it might be a better fit for this use case.

Reply:
The board revision concept does not seem to fit because these are three close yet distinct boards. Maybe it would make more sense to put all of them under the same directory, since it is already the case for the legend25_ssd and legend25_hdd boards. What do you think ?

Reply:
"Close yet distinct boards" seems like what revisions are for -- the "revision" is not the same as a "version number", and you can have custom revision handling:

https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#custom-revision-cmake-files

So you could have `legend@25ssd`, `legend@25hdd`, `legend@35`.

But maybe that won't work for you -- you know your boards better than I do. At any rate, using revisions would require putting them all in the same directory, so we seem to be going in the same direction here.

Reply:
You were right, the three boards can indeed be described using the board revision feature, I pushed a new version with these changes